webapp/src/modules/rental/reducer.spec.ts

```diff
@@ -394,3 +397,112 @@ describe('when reducing the failure action of removing a rental', () => {
     })
   })
 })
+
+describe('when reducing the action of the start of a accept rental listing', () => {
+  let periodIndexChosen: number
+  let addressOperator: string
+  beforeEach(() => {
+    periodIndexChosen = 0
+    addressOperator = '0xoperator'
+    rentalState = {
+      ...rentalState,
+      isSubmittingTransaction: false,
+      loading: [],
+      error: 'anError'
+    }
+  })
+
+  it('should set the action into loading, the submitting transaction flag as true and clear the error', () => {
+    expect(
+      rentalReducer(
+        rentalState,
+        acceptRentalListingRequest(
+          nft,
+          rental,
+          periodIndexChosen,
+          addressOperator
+        )
+      )
+    ).toEqual({
+      ...rentalState,
+      isSubmittingTransaction: true,
+      loading: [
+        acceptRentalListingRequest(
+          nft,
+          rental,
+          periodIndexChosen,
+          addressOperator
+        )
+      ],
+      error: null
+    })
+  })
+})
+
+describe('when reducing the action the success of accepting a rental', () => {
+  let periodIndexChosen: number
+  let addressOperator: string
+  beforeEach(() => {
+    periodIndexChosen = 0
+    addressOperator = '0xoperator'
+    rentalState = {
+      ...rentalState,
+      data: {},
+      isSubmittingTransaction: true,
+      loading: [
+        acceptRentalListingRequest(
+          nft,
+          rental,
+          periodIndexChosen,
+          addressOperator
+        )
+      ],
+      error: 'anError'
+    }
+  })
+
+  it('should set the submitting transaction flag as false and clear the error', () => {
+    expect(rentalReducer(rentalState, acceptRentalListingSuccess(nft))).toEqual(
+      {
+        ...rentalState,
+        data: {},
```

Comment:
If data shouldn't change, I would remove it from here and inherit it from the `rentalState` variable.
```suggestion 
        data: {},  
```